src/uwtools/config.py

```diff
@@ -331,14 +331,19 @@ def update_values(self, new_dict, dict_to_update=None):
         '''
 
         if dict_to_update is None:
-            dict_to_update = self.data
+            dict_to_update = self.data or {}
+
+        if not new_dict:
+            return
 
         for key, new_val in new_dict.items():
             if isinstance(new_val, dict):
                 if isinstance(dict_to_update.get(key), dict):
                     self.update_values(new_val, dict_to_update[key])
                 else:
                     dict_to_update[key] = new_val
+            elif new_val is None:
+                dict_to_update.pop(key, None)
```

Comment:
The standard approach in SRW is to use this mechanism to remove items that may have been set in the base file.